sky/shell/ios/sky_surface.mm

```diff
@@ -137,7 +141,24 @@ - (NSString*)skyInitialLoadURL {
 }
 
 - (NSString*)skyInitialBundleURL {
-  return [[NSBundle mainBundle] pathForResource:@"app" ofType:@"skyx"];
+  NSString *skyxBundlePath = [[NSBundle mainBundle] pathForResource:@"app" ofType:@"skyx"];
+#ifndef NDEBUG
+  NSFileManager *fileManager = [NSFileManager defaultManager];
+  NSError *error;
```

Comment:
You want to initialize this to nil. Don't think it matters since you don't call any methods on it once `copyItemAtPath:...` fails. But still.